sdk/core/azure-core/src/main/java/module-info.java

```diff
@@ -2,19 +2,17 @@
 // Licensed under the MIT License.
 
 module com.azure.core {
-    requires java.xml;
+    requires transitive reactor.core;
+    requires transitive org.reactivestreams;
+    requires transitive org.slf4j;
 
     requires transitive com.fasterxml.jackson.core;
-    requires transitive com.fasterxml.jackson.annotation;
     requires transitive com.fasterxml.jackson.databind;
+
+    requires com.fasterxml.jackson.annotation;
```

Comment:
Can you add `transitive` for jackson annotation too? Any client library requiring azure-core can also get this without having to explicitly add this in their own modules.